bitnami/minio/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/minio
   - https://min.io
-version: 12.0.3
+version: 12.1.0
```

Comment:
I would like to open a discussion on the Chart version for this PR.

On one side, MinIO deprecated the Gateway feature some versions ago, so this probably won't affect users using the latest version of the `bitnami/minio` image.

Although, for those users using an older version of the image, this change could be considered non-backward reversible, which should be included in a major version.

@corico44 @carrodher @dgomezleon What do you think? Is it fine to release this in version `12.1.0` or should be instead released as `13.0.0`?

Reply:
Sure, if backward compatibility is not guaranteed the version should be bumped in a major. I'm not familiar with this feature but if you think it could be non-backward compatible, feel free to bump the chart version in a major.

Being said that, please note bumping a major version implies bumping the major version as well in those Helm charts using MinIO as a subchart, in this case:
```console
$ ag '    name: minio' bitnami/*/Chart.yaml -l
bitnami/mastodon/Chart.yaml
bitnami/thanos/Chart.yaml
```

Reply:
But the feature was removed from 11.x.x to 12.x.x, we are only removing related variables. Am I wrong?

Reply:
Not exactly. MinIO deprecated Gateway in version `2022-10-24`, meaning that the latest version where MinIO gateway works without changing the default image version would be `11.10.12` (https://github.com/bitnami/charts/pull/13132), and that only applies to 'nas' gateway, as support for other Gateway platforms have been progressively removed in even earlier versions of MinIO.

Although, users could still run the chart up to the latest version by providing an older MinIO tag to continue using gateway feature, although that may not ensure the chart would work properly.

Reply:
After the discussion, please we decided to go ahead with version 12.1.0, with a few clarifications in the README.md